inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -422,18 +423,20 @@ void MKLDNNGraph::ExecuteConstantNodesOnly() {
         return std::make_tuple(hasExternalInvalidEdges, hasLocalAllocatedEdges, outputs);
     };
 
-    for (auto &graphNode : constantGraphNodes) {
+    for (const auto &node : constantGraphNodes) {
         if (weightsCache) {
-            auto sharedOutputs = acquireSharedOutputs(graphNode);
+            auto sharedOutputs = acquireSharedOutputs(node);
 
             if (std::get<0>(sharedOutputs) || std::get<1>(sharedOutputs)) {
-                graphNode->execute(stream);
+                DUMP(node);
+                node->execute(stream);
 
                 for (auto & output : std::get<2>(sharedOutputs))
                     output->valid(true);
             }
         } else {
-            graphNode->execute(stream);
+            DUMP(node);
```

Comment:
I propose to create "executeNode" method for graph:
```c++
inline void executeNode(node) {
        DUMP(node, infer_count);
        OV_ITT_SCOPED_TASK(itt::domains::MKLDNNPlugin, node->profiling.execute);
        node->execute(stream);
}
```
Not sure if we need ITT counters for constant node though

Reply:
Agree. Lets create. We need ITT here as well for FIL analysis